lib/loggly/common.js

```diff
@@ -165,41 +156,67 @@ common.loggly = function () {
   if (auth) {
     requestOptions.headers.authorization = 'Basic ' + new Buffer(auth.username + ':' + auth.password).toString('base64');
   }
-  if (requestBody) {
+  function popMsgsAndSend() {
+    if (isBulk) {
+      var bulk = createBulk(arrMsg);
+      sendBulkLogs(bulk);
+    } else {
+      var input = createInput(requestBody);
+      sendInputLogs(input);
+    }
+  }
+  function createBulk(msgs) {
```

Comment:
@mchaudhary In this function, I am attaching some fields with the event like attempt number, eventId and time to retry in so that later the library can identify which event was failed on which error and in what time this event needs to retry.